spinalcordtoolbox/scripts/sct_register_multimodal.py

```diff
@@ -44,7 +44,7 @@
 from spinalcordtoolbox.utils.fs import extract_fname
 from spinalcordtoolbox.image import check_dim
 
-from spinalcordtoolbox.scripts.sct_register_to_template import register_wrapper
+from spinalcordtoolbox.registration.core import register_wrapper
```

Comment:
Yes, ideally we wouldn't be importing from `scripts` (outside of `testing/` anyway). I see we only have about a dozen other instances in the codebase, which I'm sure we'll fix over time, so that's not too bad.